bin/run.sh

```diff
@@ -33,7 +33,7 @@ APPSERVICECONFIGURABLELOGSPATH=$BASEPATH/appserviceconfigurable$TIMESTAMPFORMAT.
 EDGEXLOGSPATH=$BASEPATH/edgex$TIMESTAMPFORMAT.log
 
 securityTest() {
-	$(dirname "$0")/setupSecurityserviceTest.sh
+	. $(dirname "$0")/setupSecurityserviceTest.sh
```

Comment:
why do we need the dot here?

Reply:
At some point, code to set the Kong token and Vault token was moved from runSecurity to setupSecurity.  Without the dot ("source") the environment variable will die in the subshell and won't be passed to the runSecurity script.  Basically, all of these tests have been running with invalid Kong/Vault tokens for quite some time.